### DISCUSSION TITLE
HD 238108 Missing

### DISCUSSION THREAD

**Discussion by okNimpp** [Stellarium/stellarium#3471] [General]
HD 238108 is missing. It is a part of [Messier 40](https://en.wikipedia.org/wiki/Winnecke_4) but it is missing from the Stellarium mobile app.

**Comment by alex-w** [comment]
Why you think that this star is missing?

**Comment by gzotti** [comment]
If it is missing from the mobile app, why do you report it here (desktop app)?

**Reply by okNimpp** [reply]
wait what

**Reply by okNimpp** [reply]
Where can I report an issue with the stellarium mobile app?

**Reply by gzotti** [reply]
Googling for "Stellarium mobile" finds a website with a link that says "CONTACT US".
